web-app/js/portal/filter/ui/FilterGroupPanel.js

```diff
@@ -158,8 +157,7 @@ Portal.filter.ui.FilterGroupPanel = Ext.extend(Ext.Container, {
             this.dataCollection.getFilters(),
             this.dataCollection.layerSelectionModel.selectedLayer.url,
             this.dataCollection.layerSelectionModel.selectedLayer.wmsName.split('#')[0],
-            "application/json",
-            true
```

Comment:
this "true" is needed to make some non-gridded layers work, eg anmn

Reply:
re face to face discussion, 'true' still works for me on argo with spatial extent

Reply:
ok then. I'll add it